pegasus/sites.v3/code.org/views/volunteer_engineer_form.haml

```diff
@@ -5,7 +5,10 @@
 %script{type: "text/javascript", src: "/js/microplugin.min.js"}
 %script{type: "text/javascript", src: "/js/selectize.min.js"}
 %link{rel: "stylesheet", type: "text/css", href: "/css/selectize.bootstrap3.css"}/
-%script{type: "text/javascript", src: "https://maps.googleapis.com/maps/api/js?client=#{CDO.google_maps_client_id}&sensor=false&libraries=places,geometry"}
```

Comment:
We've been trying to gate which API we use based on a flag. Is it possible to do that for this or would it be too much work? (If it's too much work, no big deal).

Reply:
Talked offline and we can bash on staging